initializr-service/src/main/resources/application.yml

```diff
@@ -123,8 +123,10 @@ initializr:
           - versionRange: "[2.0.0.RC2,2.0.0.RC2]"
             version: 2.0.0.M5
             repositories: spring-milestones
-          - versionRange: "2.0.0.RELEASE"
+          - versionRange: "[2.0.0.RELEASE,2.0.0.RELEASE]"
             version: 2.0.0.RELEASE
+          - versionRange: "2.0.1.RELEASE"
+            version: 2.0.1.RELEASE
```

Comment:
This implies that Spring Statemachine 2.0.1 incompatible with Spring Boot 2.0.0. Is that the case?

Reply:
In theory it should work but is not really tested. I've just never really understood what I need to put in these fields when upgrading versions.

Reply:
Thanks, Janne. In that case, I think this should be a one-line change to update the version from `2.0.0.RELEASE` to `2.0.1.RELEASE` and leave the range unchanged. Could you make that update please?